### ISSUE TITLE
MSA creation protocol

### ISSUE THREAD

**Issue by florisvdf** [OATML-Markslab/ProteinGym#21]
Hi Pascal,

I was wondering what protocol was used to create MSAs. Is it identical to the one used in [EVE](https://github.com/OATML-Markslab/EVE?tab=readme-ov-file)?

Best regards,
Floris

**Comment by pascalnotin** [comment]
Hi Floris,

Thanks for asking the question -- just realizing we did not include these details in the latest version of the paper!
We followed a process similar to the one described in the [Tranception paper](https://proceedings.mlr.press/v162/notin22a.html) (see Appendix C). In summary: we ran Jackhmmer on UniRef100, exploring various bit scores (from 0.1 to 0.9) and selected the alignment with the highest number of significant Evolutionary Couplings (using the [EV couplings](https://github.com/debbiemarkslab/EVcouplings) pipeline). 
For each assay, we aimed to generate an alignment for the full protein sequence. This helped ensure for instance that alignment-based methods would be able to score all assayed mutants. In a handful of cases this was impractical (e.g., very long sequences), and we instead created alignments for the mutated region (with a 10AA-long padding on both sides of the mutated region).

Best,
Pascal
